### PR TITLE
Add warning about fastcomp deprecation

### PR DESCRIPTION
**Pull request by sbc100**
Can be disabled, like all warnings, with `-Wno-fastcomp`

### CODE REVIEW

File: emcc.py [hunk 1]
```diff
@@ -1258,6 +1258,9 @@ def get_last_setting_change(setting):
 
     shared.verify_settings()
 
+    if not shared.Settings.WASM_BACKEND:
+      diagnostics.warning('fastcomp', 'the fastomp compiler will soon be deprecated.  Please switch to the upstream llvm backend as soon as possible and open issues if you have trouble doing so')
```
Comment (dschuff) [inline emcc.py:1262]: Can we say that it *is* deprecated and will soon be removed? or soon stop being updated?

Comment (kripken) [reply emcc.py:1262]: maybe saying something about when it's removed/no longer updated should wait for a mailing list discussion?

Comment (kripken) [reply emcc.py:1262]: But +1 to saying it is deprecated already, which we've been saying for a while.

Comment (sbc100) [reply emcc.py]: Done

**Comment by sbc100** [comment]
I think i'm happy with message as is now..  No need to embed the entire plan in the error string.   "

But this did make be realize I don't think we have bug for removing fastcomp :)

**Pull request merged**